app/javascript/src/controller_services.js

```diff
@@ -217,54 +208,50 @@ class PageAdditionMenu extends ActivatedMenu {
  * 3). Close the open menu
  * 4). Open the form URL input dialog.
  **/
-PageAdditionMenu.selection = function(event, data) {
-  var $activator = data.activator.find("> a");
-  var form = this.dialog.$form; // Form sending information back to server.
+PageAdditionMenu.selection = function (event, data) {
+  const $activator = data.activator.find("> a")
+  const form = this.dialog.$form // Form sending information back to server.
 
   // First reset to remove any lingering values.
-  updateHiddenInputOnForm(form, "page[page_type]", "");
-  updateHiddenInputOnForm(form, "page[component_type]", "");
+  updateHiddenInputOnForm(form, "page[page_type]", "")
+  updateHiddenInputOnForm(form, "page[component_type]", "")
 
   // Then add any required values.
-  if($activator.length) {
-    updateHiddenInputOnForm(form, "page[page_type]", $activator.data("page-type"));
-    updateHiddenInputOnForm(form, "page[component_type]", $activator.data("component-type"));
+  if ($activator.length) {
+    updateHiddenInputOnForm(form, "page[page_type]", $activator.data("page-type"))
+    updateHiddenInputOnForm(form, "page[component_type]", $activator.data("component-type"))
 
-    data.component.close();
-    $("#new-page-create-dialog").dialog("open");
+    data.component.close()
+    $("#new-page-create-dialog").dialog("open")
   }
 }
 
-
 /* Quickfix workaround to try and fix scrolling issues for the form overview
  * when there are too many thumbnails to fix on the one page view.
  **/
-function applyCustomOverviewWorkaround() {
-  var $overview = $("#form-overview");
-  var $container = $overview.find(" > .container");
-  var $button = $(".form-overview_button")
-  var containerWidth = $container.width();
-  var overviewWidth = $overview.width();
-  var offsetLeft = $overview.offset().left;
-  var margin = 30; // Arbitrary number based on common
-  var spacerForMenu = 250;
-  var maxWidth = window.innerWidth - (margin * 2) - spacerForMenu;
-
-  if(containerWidth > overviewWidth) {
-    let left = ((containerWidth + spacerForMenu) - overviewWidth) / 2;
-    if(left < offsetLeft) {
-      $container.css("left", ~left);
-    }
-    else {
-      $container.css("left", ~(offsetLeft - margin));
+function applyCustomOverviewWorkaround () {
+  const $overview = $("#form-overview")
+  const $container = $overview.find(" > .container")
+  const $button = $(".form-overview_button")
+  const containerWidth = $container.width()
+  const overviewWidth = $overview.width()
+  const offsetLeft = $overview.offset().left
+  const margin = 30 // Arbitrary number based on common
+  const spacerForMenu = 250
+  const maxWidth = window.innerWidth - (margin * 2) - spacerForMenu
+
+  if (containerWidth > overviewWidth) {
+    const left = ((containerWidth + spacerForMenu) - overviewWidth) / 2
+    if (left < offsetLeft) {
+      $container.css("left", ~left)
+    } else {
+      $container.css("left", ~(offsetLeft - margin))
```

Comment:
Again, yes, we are probably declaring variable values that are not actually being changed within the scope they are used here, thus allowing a robot to exercise it's programmed prejudice against the `var` keyword and replace with `const`, but that also implies these values should not/cannot be changed.

It might be correct for today, or this small scope of code, but what happens when some change is required that means...
`const left = (some calculation here)`
...needs to be changed to include something like
```
else {
  left = (some other calculation)
}
```
Some poor developer is likely to be wondering _if/can I/should I_ about a simply update, perhaps more than necessary.

Guess I just don't like seeing `const` all over the place.